common/datablocks/src/data_block.rs

```diff
@@ -34,10 +34,16 @@ pub struct DataBlock {
 impl DataBlock {
     #[inline]
     pub fn create(schema: DataSchemaRef, columns: Vec<ColumnRef>) -> Self {
-        debug_assert!(schema.fields().iter().zip(columns.iter()).all(|(f, c)| {
-            f.data_type().data_type_id().to_physical_type()
-                == c.data_type().data_type_id().to_physical_type()
-        }));
+        debug_assert!(
+            schema.fields().iter().zip(columns.iter()).all(|(f, c)| f
+                .data_type()
+                .data_type_id()
+                .to_physical_type()
+                == c.data_type().data_type_id().to_physical_type()),
+            "Schema: {:?}, columns: {:?}",
```

Comment:
`columns` will also print all `DataValue`s? If so when the amount of data is very large, it brings noise.

Reply:
It's only for debug, I think this is fine.

Reply:
Fixed.